communications/slack-culture.rst

```diff
@@ -45,28 +45,103 @@ Summary
+They are typically named after the cluster whose alerts they receive (eg ``status-data-lsst-cloud`` for alerts relevant to data.lsst.cloud).
+Although sometimes operational staff will comment on alerts in those channels to annotate events, these channels should be considered read-only for all practical purposes.
+
+Prepend ``status-`` to the channel's name for channels of this nature.
+
+Summit Channels
+---------------
+
+Channels oriented towards summit activities, including daywork and night observing, have their own set of protocols as they carry communication of a critical nature, involve personnel that are often off-line/off-shift and cannot be assumed to have read all traffic, and are used by staff that are aggressively multitasking and monitoring multiple communication channels.
+
+If you are involved in summit work, make sure you clearly understand the rules of engagements of those channels, when and how to post, and (if relevant to your duties) how to respond to support requests.
+
+Summit slack communication protocols are described in the `Observatory Operations guide <https://obs-ops.lsst.io/Daytime-Nighttime-Interactions/slack-channel-usage.html>`_.
+
+Note that not all summit-type channels are prepended by ``summit-`` although some are.
+
+Support Channels
+----------------
+
+Support channels are intended as primary support venues for obtaining technical assistance that you require in order to do your work.
+Support channels may have different arrangements for how and when problems are reported to them and what to expect from a response.
+Consult the (:ref:`Support Channels<slack-support-channels>`) section for more guidance.
 
-Support channels are ones in which users of a service, software package, or dataset interact with developers, primarily by asking questions and reporting problems.
-These are often but not always indicated in the channel name with "-support", "-users", "-problems", "-ask", or "help-".
+Append ``-support`` to the channel name, and name the channel carefully to minimize misdirected questions (if you're always telling people they are asking in the wrong channel, that's a clue).
 
-Informal channels are ones for interactions that are not at all or not directly work-related.
-They are often location-specific or oriented towards particular interests or hobbies.
+Note: there is an emerging practice to reserve the ``-support`` suffix for channels where there is some kind of formal undertaking to provide, well, support.
+For more informal, unofficial and peer-to-peer Q&A channels, consider using ``-help`` instead.
 
-Discussion channels are all the rest: work-related discussions about particular topics.
+Team Channels
+-------------
 
-We are looking into renaming channels to make their type more obvious.
+If your team has a "find us here" channel, append ``-team`` to the channel name.
+
+Again, avoid unnecessary taxonomy: it turns out ``square-team`` is a more useful name to people that ``dm-square``.
+
```

Comment:
We have typically used `lt` for a leadership team - any guidance on this? #dmlt #splt? or #sp-lteam ?